notifications_api_common/exponential_backoff.py

```diff
@@ -0,0 +1,15 @@
+import random
+
+
+def get_exponential_backoff_interval(
```

Comment:
Might be good to add some simple unit tests to verify that this calculates the delay correctly (with different base factors)